node/src/main/scala/org/bitcoins/node/NeutrinoNode.scala

```diff
@@ -19,7 +19,7 @@ import scala.concurrent.Future
 
 case class NeutrinoNode(
     nodePeer: Vector[Peer],
-    dataMessageHandler: DataMessageHandler,
+    var dataMessageHandler: DataMessageHandler,
```

Comment:
What was the reason for changing this? This makes `dataMessageHandler` a publicly available mutable variable which is generally a bad idea.

Reply:
The reason for changing this here is for changing `updateDataMessageHandler` to what it is now. Earlier used `copy` to update and but did not copy the vals defined in `Node` trait that `NeutrinoNode` extends so this led to the `peerServices` map we had in `Node` trait to be initialized to empty. Also, there was one other place where states diverged in `DataMessageHandler` and led to errors in `handleDataPayload` of  `PeerMessageReceiver`. 

I get that this should not be public so made it private and added a getter for it.